staging/src/k8s.io/apimachinery/pkg/util/proxy/upgradeaware.go

```diff
@@ -230,9 +232,22 @@ func (h *UpgradeAwareHandler) ServeHTTP(w http.ResponseWriter, req *http.Request
 	proxy := httputil.NewSingleHostReverseProxy(&url.URL{Scheme: h.Location.Scheme, Host: h.Location.Host})
 	proxy.Transport = h.Transport
 	proxy.FlushInterval = h.FlushInterval
+	proxy.ErrorLog = log.New(noSuppressPanicError{}, "", log.LstdFlags)
 	proxy.ServeHTTP(w, newReq)
 }
 
+type noSuppressPanicError struct{}
+
+func (noSuppressPanicError) Write(p []byte) (n int, err error) {
+	// skip "suppressing panic for copyResponse error in test; copy error" error message
+	// that ends up in CI tests on each kube-apiserver termination as noise and
+	// everybody thinks this is fatal.
+	if strings.Contains(string(p), "suppressing panic") {
```

Comment:
can we limit this to tests, or only to termination time, rather than pipe all server logs through a substring check?

Reply:
This logger is used for log 3 static strings in the reverse proxy code. There is no high load.

This warning is noise in every log. T

Reply:
If you are concerned about complexity (string containment is super fast, assuming Golang devs have read Knuth), we can change it into a prefix check. But as this directly prints to stderr normally, without any verbosity check, I think a string check is better than any IO performance-wise.

Reply:
ah, didn't realize the context. I thought this was used for more. ugly, but so is a lot of detailed integration with the http stdlib :-/